packages/svelte2tsx/svelte-jsx.d.ts

```diff
@@ -198,6 +198,10 @@
 
       // Transition Events
       ontransitionend?: TransitionEventHandler<T>;
+      onoutrostart?: TransitionEventHandler<T>;
```

Comment:
These events are svelte's custom events. Can you separate them with the vanilla dom transition event with another comment? 
Also, I tried it in the [REPL](https://svelte.dev/repl/215f3b5951a7413e845b2a11642e797f?version=3.24.0), the type of event handlers should be `EventHandler<CustomEvent<null>, T>`

Reply:
like this:?
```ts
      // Vanilla Events     
      onoutrostart?: EventHandler<CustomEvent<null>, T>;
      onoutroend?: EventHandler<CustomEvent<null>, T>;
      onintrostart?: EventHandler<CustomEvent<null>, T>;
      onintroend?: EventHandler<CustomEvent<null>, T>;
```

Reply:
`// Svelte Transition Events` would be better

Reply:
ok, i made the changes.

Reply:
vanilla events seem like vanilla js events to me

Reply:
```
      // Svelte Transition Events
      onoutrostart?: EventHandler<CustomEvent<null>, T>;
      onoutroend?: EventHandler<CustomEvent<null>, T>;
      onintrostart?: EventHandler<CustomEvent<null>, T>;
      onintroend?: EventHandler<CustomEvent<null>, T>;

```